contact.html

```diff
@@ -11,16 +11,16 @@
 		</p>
 		<ul>
 			<li>
-			  <a href="mailto: users@kafka.apache.org">users@kafka.apache.org</a>: A list for general user questions about Kafka&reg;. To subscribe, send an email to <a href="mailto: users-subscribe@kafka.apache.org">users-subscribe@kafka.apache.org</a>. Archives are available <a href="http://mail-archives.apache.org/mod_mbox/kafka-users">here</a>.
+				<b>User mailing list</b>: A list for general user questions about Kafka&reg;. To subscribe, send an email to <a href="mailto: users-subscribe@kafka.apache.org">users-subscribe@kafka.apache.org</a>. Once subscribed, send your emails to <a href="mailto: users@kafka.apache.org">users@kafka.apache.org</a>. Archives are available <a href="http://mail-archives.apache.org/mod_mbox/kafka-users">here</a>.
 			</li>
 			<li>
-			  <a href="mailto: dev@kafka.apache.org">dev@kafka.apache.org</a>: A list for discussion on Kafka development. To subscribe, send an email to <a href="mailto: dev-subscribe@kafka.apache.org">dev-subscribe@kafka.apache.org</a>. Archives available <a href="http://mail-archives.apache.org/mod_mbox/kafka-dev">here</a>.
+				<b>Developer mailing list</b>: A list for discussion on Kafka&reg; development. To subscribe, send an email to <a href="mailto: dev-subscribe@kafka.apache.org">dev-subscribe@kafka.apache.org</a>. Once subscribed, send your emails to <a href="mailto: dev@kafka.apache.org">dev@kafka.apache.org</a>. Archives are available <a href="http://mail-archives.apache.org/mod_mbox/kafka-dev">here</a>.
 			</li>
 			<li>
-			  <a href="mailto: jira@kafka.apache.org">jira@kafka.apache.org</a>: A list to track Kafka <a href="https://issues.apache.org/jira/projects/KAFKA">JIRA</a> notifications. To subscribe, send an email to <a href="mailto: jira-subscribe@kafka.apache.org">jira-subscribe@kafka.apache.org</a>. Archives available <a href="http://mail-archives.apache.org/mod_mbox/kafka-jira">here</a>.
+				<b>JIRA mailing list</b>: A list to track Kafka&reg; <a href="https://issues.apache.org/jira/projects/KAFKA">JIRA</a> notifications. To subscribe, send an email to <a href="mailto: jira-subscribe@kafka.apache.org">jira-subscribe@kafka.apache.org</a>. Once subscribed, send your emails to <a href="mailto: jira@kafka.apache.org">jira@kafka.apache.org</a>. Archives are available <a href="http://mail-archives.apache.org/mod_mbox/kafka-jira">here</a>.
```

Comment:
"Once subscribed, send your emails to..." I'm not sure that's how the JIRA list is intended to work. It's simply about notification of changes in JIRA; people won't normally mail the list themselves.

Reply:
Fixed.